### PR TITLE
remove search box for study cards

### LINKED ISSUE #20: Homepage header updates for site search
This ticket is to continue the work indicated in https://github.com/VEuPathDB/web-eda/issues/769

Remaining task is to change placement of tagline and drop down menus according to mock-ups on slide 7 of this doc: https://docs.google.com/presentation/d/10DBhyRYbd3q6LEYZtiungGSVJ1nncir6llsTzLPyeH0/edit#slide=id.p

This can be done at the same time as the addition of the site search finder box

### PR DESCRIPTION
**Pull request by dmfalke**
fixes #20 

This PR removes the search box for study cards

### CODE REVIEW

File: Site/package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "veupathdb-react-scripts run-site-dev-server --config webpack.config.local.mjs --port 3000"
+    "start": "veupathdb-react-scripts run-site-dev-server --config webpack.config.local.mjs --port 3000",
+    "prepare": "relative-deps"
```
Comment (jtlong3rd) [inline Site/package.json:8]: How exciting! Is this safe to merge now, or will we need to give `EbrcWebsiteCommon` a `prepare` or `postinstall` script first?

Comment (dmfalke) [reply Site/package.json:8]: Oh my, I didn't really mean to include that, but maybe I should?

Comment (dmfalke) [reply Site/package.json:8]: It uses the `build` script of the dependency, so it works in this case.

Comment (jtlong3rd) [reply Site/package.json:8]: Perfect! I say merge away.

Comment (dmfalke) [reply Site/package.json:8]: This does have implications for website builds. I think we should wait until web-common is removed from project_home.

Comment (jtlong3rd) [reply Site/package.json:8]: Sounds like a plan

Comment (dmfalke) [reply Site/package.json:8]: I will remove this specific script, but leave the rest of the changes related to relative-deps.

**Pull request merged**